packages/wct-mocha/src/reporters/multi.ts

```diff
@@ -92,12 +92,10 @@ export default class MultiReporter implements Reporter {
   }
 
   /**
-   * @param location The location this reporter represents.
    * @return A reporter-like "class" for each child suite
    *     that should be passed to `mocha.run`.
    */
-  childReporter(location: Location|string): ReporterFactory {
-    const name = this.suiteTitle(location);
```

Comment:
So this was straight up not used?  Huh.  Good find.

Reply:
I just turned on warning for unused vars and props.

That's also why I made some properties not private, because they weren't used anywhere, but I wasn't confident enough to delete them, so they're public for now.